src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -875,6 +875,7 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
             {info.vjsout[0], info.vjsout[1]};
     boost::array<size_t, ZC_NUM_JS_INPUTS> inputMap;
     boost::array<size_t, ZC_NUM_JS_OUTPUTS> outputMap;
+    std::function<int(int)> emptyFunc;
```

Comment:
In that case, I would prefer that you just not call `JSDescription::Randomized` at all, and instead adjust `find_output` to always return the given query input.